File: src/main/java/gargoyle/netsquares/util/log/LoggerFactory.java
package gargoyle.netsquares.util.log;

public final class LoggerFactory {
    public static Logger getLogger(Class<?> aClass) {
        return getLogger(aClass.getName());
    }

    public static Logger getLogger(String name) {
        return new JULLogger(name);
    }
}

File: src/main/kotlin/gargoyle/netsquares/util/log/LoggerFactory.kt
package gargoyle.netsquares.util.log

import kotlin.reflect.KClass

object LoggerFactory {
    fun getLogger(aClass: KClass<*>): Logger {
        return getLogger(aClass.qualifiedName!!)
    }

    fun getLogger(name: String): Logger {
        return JULLogger(name)
    }
}
